### PR TITLE
XAMARIN-43420 : UG content for OpenIconImageSource and CloseIconImageSource in SfBackdropPage

### PR DESCRIPTION
**Pull request by Shalini-Ashokan**
Hi Team,

To include UG content for OpenIconImageSource and CloseIconImageSource in SfBackdropPage.

Regards,
Shalini.

### CODE REVIEW

File: Xamarin/Backdrop-Page/Header-Configuration.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ When the backdrop page placed in the [MasterDetailPage](https://docs.microsoft.c
 
 ### Custom icons
 
-You can customize the default icons in the navigation header by setting the OpenIcon and CloseIcon properties in SfBackdropPage.
+You can customize the default icons in the navigation header by setting the OpenIconImageSource and CloseIconImageSource properties in SfBackdropPage.You may also change the image source formats FileImageSource, UriImageSource, FontImageSource, and StreamImageSource.
```
Comment (vigneshrameshsync) [inline Xamarin/Backdrop-Page/Header-Configuration.md:51]: It accepts following inputs
- FileImageSource
- UriImageSource
- FontImageSource
- StreamImageSource

File: Xamarin/Backdrop-Page/Header-Configuration.md [hunk 2]
```diff
@@ -86,5 +86,4 @@ namespace BackdropGettingStarted
 
 {% endtabs %}
 
- ![Open Custom icons](Header-Configuration-images/Settings.png)
- 
+ ![Open Custom icons](Header-Configuration-images/CustomIconFile.png)
```
Comment (vigneshrameshsync) [inline Xamarin/Backdrop-Page/Header-Configuration.md:89]: Don't change the image.

File: Xamarin/Backdrop-Page/Header-Configuration.md (hunk 1 above)
Comment (vigneshrameshsync) [reply Xamarin/Backdrop-Page/Header-Configuration.md:51]: Add note for UriImageSource will not work in iOS platform.

**Review by vigneshrameshsync — CHANGES_REQUESTED** [review]

File: Xamarin/Backdrop-Page/Header-Configuration.md (hunk 1 above)
Comment (Shalini-Ashokan) [reply Xamarin/Backdrop-Page/Header-Configuration.md]: Updated a changes sir

File: Xamarin/Backdrop-Page/Header-Configuration.md [hunk 3]
```diff
@@ -48,7 +48,13 @@ When the backdrop page placed in the [MasterDetailPage](https://docs.microsoft.c
 
 ### Custom icons
 
-You can customize the default icons in the navigation header by setting the OpenIcon and CloseIcon properties in SfBackdropPage.
+You can customize the default icons in the navigation header by setting the [`OpenIconImageSource`]() and [`CloseIconImageSource`]() properties in [`SfBackdropPage`](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.Backdrop.SfBackdropPage.html). It accepts following inputs:
```
Comment (Chrispineagunja) [inline Xamarin/Backdrop-Page/Header-Configuration.md:51]: Customize the default icons in the navigation header by setting the [`OpenIconImageSource`]() and [`CloseIconImageSource`]() properties in the [`SfBackdropPage`](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.Backdrop.SfBackdropPage.html). It accepts the following inputs:

Comment (Shalini-Ashokan) [reply Xamarin/Backdrop-Page/Header-Configuration.md]: Modified

**Review by vigneshrameshsync — APPROVED** [review]
The content is fine.

**Pull request merged**